app/models/course_user.rb

```diff
@@ -1,4 +1,5 @@
 class CourseUser < ActiveRecord::Base
+  include CourseUser::LevelProgressConcern
```

Comment:
Actually maybe this should be after all the declarations, does that make more sense?